Spigot-API-Patches/0296-Add-PufferFishFlateEvent.patch

```diff
@@ -0,0 +1,91 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: HexedHero <6012891+HexedHero@users.noreply.github.com>
+Date: Mon, 10 May 2021 16:58:38 +0100
+Subject: [PATCH] Add PufferFishFlateEvent
+
+
+diff --git a/src/main/java/io/papermc/paper/event/entity/PufferFishFlateEvent.java b/src/main/java/io/papermc/paper/event/entity/PufferFishFlateEvent.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..a512df6840bb7502cee9aaa4d0902b4dd8f698fd
+--- /dev/null
++++ b/src/main/java/io/papermc/paper/event/entity/PufferFishFlateEvent.java
+@@ -0,0 +1,79 @@
++package io.papermc.paper.event.entity;
++
++import org.bukkit.entity.PufferFish;
++import org.bukkit.event.Cancellable;
++import org.bukkit.event.HandlerList;
++import org.bukkit.event.entity.EntityEvent;
++import org.jetbrains.annotations.NotNull;
++
++/**
++ * Called just before a {@link PufferFish} inflates or deflates.
++ */
++public class PufferFishFlateEvent extends EntityEvent implements Cancellable {
++    private static final HandlerList handlers = new HandlerList();
++    private boolean cancelled;
++    private int newPuffState;
++
++    public PufferFishFlateEvent(@NotNull PufferFish entity, int newPuffState) {
++        super(entity);
++    }
++
++    @NotNull
++    @Override
++    public PufferFish getEntity() {
++        return (PufferFish) entity;
++    }
++
++    /**
++     * Get the <strong>new</strong> puff state of the {@link PufferFish}.
++     * <br><br>
```

Comment:
I believe Paper prefers `<p>` blocks over `<br><br>`: I was asked to change something like this once.